parm/config/gfs/config.resources

```diff
@@ -59,7 +59,15 @@ case ${machine} in
     ;;
   "AWSPW")
     export PARTITION_BATCH="compute"
-    npe_node_max=40
+    npe_node_max=36
+    ;;
+  "AZUREPW")
+    export PARTITION_BATCH="compute"
+    npe_node_max=24
+    ;;
+  "GOOGLEPW")
+    export PARTITION_BATCH="compute"
+    npe_node_max=30
```

Comment:
Please withhold till these platforms are available and the workflow is being ported to them.